core/src/test/scala/org/apache/spark/sql/expression/Aggregate0Suite.scala

```diff
@@ -0,0 +1,161 @@
+/*
+ *
+ * Copyright 2017 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.spark.sql.expression
+
+import org.apache.spark.sql.BaseTiSparkSuite
+import org.apache.spark.sql.test.SharedSQLContext
+
+class Aggregate0Suite extends BaseTiSparkSuite with SharedSQLContext {
+
+  test("select tp_double from full_data_type_table  group by (tp_double)  order by tp_double ") {
+    runTest(
```

Comment:
can we wrap runTest() into a single function, e.g., runTestAll() ?

Reply:
Something like https://github.com/pingcap/tispark/pull/127/files#diff-604c956654a282408905e45528650602 ArithmeticTest2Suite.scala ?

Reply:
yeah, we can wrap `runTest(query, query.replace("full_data_type_table", "full_data_type_table_j"))` into a single function.
Maybe that would be too much work? LOL

Reply:
Emmmm some work will do.....these generated code could be improved.

Reply:
Done.